Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  XPCMachServer.swift
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-08.
+//
+
+import Foundation
+
+internal class XPCMachServer: XPCServer {
+	private let machService: xpc_connection_t
+	private let clientRequirements: [SecRequirement]
+
+	internal init(machServiceName: String, clientRequirements: [SecRequirement]) {
+		self.clientRequirements = clientRequirements
+
+		self.machService = machServiceName.withCString { serviceNamePointer in
+			return xpc_connection_create_mach_service(
+				serviceNamePointer,
+				nil, // targetq: DispatchQueue, defaults to using DISPATCH_TARGET_QUEUE_DEFAULT
+				UInt64(XPC_CONNECTION_MACH_SERVICE_LISTENER))
+		}
+	}
+
+	internal static func _forBlessedHelperTool() throws -> XPCMachServer {
```

Comment:
I understand why the the leading `_` is here, but it seems rather un-Swift like? Maybe call it `createForBlessedHelperTool()`? I'm not horribly opposed to the underscore, it just looks odd to me.

Reply:
I needed some arbitrary distinction to make it not clash with the inherited `forBlessedHelperTool()`.

Alternatively, I could make the parent as `class` (instead of `static`, which is like `final class`), but I didn't want that to be exposed to public API